gateway/main/settings.py

```diff
@@ -89,17 +92,40 @@
 
 WSGI_APPLICATION = "main.wsgi.application"
 
+LOGGING = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "handlers": {
+        "console": {
+            "class": "logging.StreamHandler",
+        },
+    },
+    "root": {
+        "handlers": ["console"],
+        "level": "DEBUG",
+    },
+}
 
 # Database
 # https://docs.djangoproject.com/en/4.1/ref/settings/#databases
 
 DATABASES = {
     "default": {
+        "ENGINE": "django.db.backends.postgresql",
+        "NAME": os.environ.get("DATABASE_NAME", "bitnami_keycloak"),
+        "USER": os.environ.get("DATABASE_USER", "bn_keycloak"),
+        "PASSWORD": os.environ.get("DATABASE_PASSWORD", "wHXJPZSOGV"),
+        "HOST": os.environ.get("DATABASE_HOST", "main-postgresql"),
+        "PORT": os.environ.get("DATABASE_PORT", "5432"),
+    },
+    "test": {
         "ENGINE": "django_prometheus.db.backends.sqlite3",
         "NAME": BASE_DIR / "db.sqlite3",
-    }
+    },
 }
 
+if "test" in sys.argv:
```

Comment:
Just as a doubt @akihikokuroda , now if we want to use `sqlite` what command do we need to use: `python manage.py runserver test`?

Reply:
It doesn't give the choice to use `sqlite` for runtime.  It is for using the sqlite for the unit tests.